templates/Menus/nas-main.html.twig

```diff
@@ -0,0 +1,49 @@
+                    <li class="middle even menuparent"><span class="nav-chart menuparent nolink sf-with-ul">Tracking &amp; Stats<span class="sf-sub-indicator"> »</span><span class="sf-sub-indicator"> »</span></span>
+                        <ul class="sf-hidden" style="float: none; width: 12em;">
+                            <li class="first odd "><a href="{{  path('pelagos_app_ui_datasetmonitoring_default')  }}">Dataset monitoring</a></li>
+                            <li class="last even "><a href="{{  path('pelagos_app_ui_stats_default')  }}">GRIIDC system statistics</a></li>
+                        </ul>
+                    </li>
+                    <li class="middle odd menuparent"><span class="menuparent nolink sf-with-ul">About Us<span class="sf-sub-indicator"> »</span><span class="sf-sub-indicator"> »</span></span>
+                        <ul class="sf-hidden" style="float: none; width: 12em;">
+                            <li class="first odd "><a href="/about-griidc">About GRIIDC</a></li>
+                            <li class="middle even sf-depth-2"><a href="/meet-the-team">Meet the team</a></li>
+                            <li class="middle odd "><a href="/griidc-services">GRIIDC Services</a></li>
+                            <li class="last even "><a href="/terms-and-conditions">Terms &amp; Conditions</a></li>
+                        </ul></li>
+                    <li class="middle even"><span class="menuparent nolink sf-with-ul">Help<span class="sf-sub-indicator"> »</span>
+                                            <span class="sf-sub-indicator"> »</span></span>
+                        <ul class="sf-hidden" style="float: none; width: 13.5em;">
+                            <li class="first odd "><a href="{{ path('pelagos_app_ui_account_default')  }}">Account Request</a></li>
+                            <li class="middle even "><a href="{{ path('pelagos_app_ui_account_password')  }}">Password Reset</a></li>
+                            <li class="middle odd "><a href="{{ path('pelagos_app_ui_account_forgotusername') }}">Forgot Username</a></li>
+                            <li class="middle even "><a href="/frequently-asked-questions">FAQs</a></li>
+                            <li class="middle odd "><a href="/data-management-planning">Data Management Planning</a></li>
+                            <li class="middle even "><a href="/training-user-guides">Training &amp; User Guides</a></li>
+                            <li class="middle odd "><a href="/video-tutorials">Video Tutorials</a></li>
+                            <li class="middle even "><a href="/data-file-transfer-methods">Data File Transfer Methods</a></li>
+                            <li class="middle odd "><a href="/metadata-editor-start">ISO 19115-2 Metadata Editor</a></li>
+                        </ul>
+                    </li>
```

Comment:
These classes: First, Odd, Even, Last don't seem to do anything...
That should make it easier to add the Admin menu here.